docs/api-usage.rst

```diff
@@ -238,6 +238,28 @@ a project (the previous example used 2 API calls):
    project = gl.projects.get(1, lazy=True)  # no API call
    project.star()  # API call
 
+``head()`` methods
+========================
+
+All endpoints that support ``get()`` and ``list()`` also support a ``head()`` method.
```

Comment:
I didn't see the mixin being added to the `list` mixin. Is this documentation correct?

Reply:
Oh damn :see_no_evil: Nice catch, thanks for that. I've added it now :)